### PR TITLE
Modify ingestfile gui

### PR DESCRIPTION
**Pull request by privatezero**
Adds option in GUI to perform interlacement test.  If option not selected there is no test or subsequent prompt.

### CODE REVIEW

File: ingestfile [hunk 1]
```diff
@@ -166,6 +166,7 @@ FORMULA.tooltip = To force the input file to be interpreted as top or bottom fie
 FORMULA.default = none
 FORMULA.option =
 FORMULA.option = none
+FORMULA.option = Perform Test
```
Comment (dericed) [inline ingestfile]: `Perform Test` is a little vague, how about `check interlacement`?

Comment (privatezero) [inline ingestfile]: makes sense!!

**Comment by ablwr** [comment]
it makes *sensei*!

**Comment by privatezero** [comment]
hahaha very _punny_

**Pull request merged**